templates/contribute/index.md

```diff
@@ -45,7 +46,8 @@ Typical workflow:
   ```
 * Sometimes you may not want to create a new branch, but instead work on a branch
```

Comment:
I'm not sure we should keep this paragraph: if you start from someone else's branch like that, you won't be able to push back the changes, contrary to what is explained in the next paragraphs. Or add that, after the `gh pr checkout 12345`, you should also do `git switch -c my_copy_of_the_branch` to be able to push back (but to a branch on your fork, not the original one)?

Reply:
I share this doubt. Replacing it by a pointer to the other docs would be equally sensible.